.github/pull_request_template.md

```diff
@@ -0,0 +1,21 @@
+<!-- This template is for Devs to give QA details before moving the issue To-Test -->
+
+<!-- Link to the issue -->
+
+### Root cause
+<!-- Define findings related to the feature or bug issue. -->
+
+### What was fixed, or what changes have occurred
```

Comment:
we could also add these tick boxes

```
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Upgrade (Upgrade dependencies, support, versions)

<!-- Include information of the changes, including collateral areas which have been affected by this PR as requirement or for convenience. -->
```

Reply:
Shouldn't this information be easily extracted from the linked issue?

Reply:
I was thinking more about rely on the labels, unless we want to create different templates.